index.js

```diff
@@ -34,10 +22,16 @@ module.exports = {
       parent.import(path.join('vendor', 'fonts', fontFileName));
     });
 
-    importFixtable(parent);
+    let fixtablePath = path.join(this.nodeModulesPath, 'fixtable/dist/');
+
+    parent.import({
+      development: path.join(fixtablePath, 'fixtable.js'),
+      production: path.join(fixtablePath, 'fixtable.min.js')
```

Comment:
Since we're moving this into our build tree, do we need to include the minimized JS file here, or should we just include the full thing?